Vagrantfile

```diff
@@ -80,6 +80,13 @@ $provision_script = <<SCRIPT
 set -x
 set -e
 set -o pipefail
+# If the host is running SELinux remount the /sys/fs/selinux directory as read only,
+# needed for apt-get to work.
+if [ -d "/sys/fs/selinux" ]; then
+  sudo mount -o remount,ro /sys/fs/selinux
+fi
+# Set default locale, this prevents errors on vagrant ssh
+echo "LC_ALL=en_US.UTF-8" | sudo tee -a /etc/default/locale
```

Comment:
I was away for Thanksgiving holiday, sorry for the delay.
The LC_ALL problem is as far as I understand the following:
- My host stystems locale:
```LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=nl_NL.UTF-8
LC_TIME=nl_NL.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=nl_NL.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=nl_NL.UTF-8
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT=nl_NL.UTF-8
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```
The vagrant box locale if I do not set `LC_ALL when sshing`:
```
locale: Cannot set LC_ALL to default locale: No such file or directory
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=nl_NL.UTF-8
LC_TIME=nl_NL.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=nl_NL.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=nl_NL.UTF-8
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT=nl_NL.UTF-8
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```
And now programs like perl complain loudly that they are having trouble with the locale. My guess is because in my example nl_NL.UTF-8 is not installed on the vagrant box.
To solve this we could 
- install te locale from the host system on the vagrant system;
- or force the vagrant system to always use en_US.UTF-8 (by setting LC_ALL.

What I was doing is the second step as it seams simpler and makes sure the vagrant systems are not different for different devs.

I'll also look into `update-locale`